### PR TITLE
Docs revise overview supported-operations.md

### PR DESCRIPTION
**Pull request by seanshpark**
on going draft to revise overview/supported-operations.md document

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #2654

**Comment by seanshpark** [comment]
@lemmaa PTAL

File: docs/overview/supported-operations.md [hunk 1]
```diff
@@ -1,7 +1,143 @@
 # Supported Operations
 
-## Compiler
+As of 2020-06-26
+- TensorFlow commit e5023a1738cce7efcdf9d87863b85c80ab2f8c9e
```
Comment (lemmaa) [inline docs/overview/supported-operations.md:4]: I would like to add a rough version information as well as a commit hash. For example 'Tensorflow nightly build after v2.2.0 release.' , etc.

File: docs/overview/supported-operations.md [hunk 2]
```diff
@@ -1,7 +1,143 @@
...
+TOPK_V2 | O | O |  
+TRANSPOSE | O | O |  
+TRANSPOSE_CONV | O | O |  
+UNIDIRECTIONAL_SEQUENCE_LSTM | O |   |  
+UNIDIRECTIONAL_SEQUENCE_RNN | O |   |  
+UNIQUE | O |   |  
+UNPACK | O | O |  
+WHERE | O |   |  
+WHILE | O | O |  
+ZEROS_LIKE | O | O |
```
Comment (lemmaa) [inline docs/overview/supported-operations.md:135]: How about adding the sum of O of each column to the bottom of the table? Is optional. :)

**Comment by seanshpark** [comment]
@hseok-oh ping

**Comment by hseok-oh** [comment]
Runtime (supporting at least one of backend on this repo)

- ADD_N 
- ARG_MIN
- BATCH_TO_SPACE_ND
- BIDIRECTIONAL_SEQUENCE_LSTM
- BIDIRECTIONAL_SEQUENCE_RNN
- CALL
- CEIL
- CONCAT_EMBEDDINGS
- DELEGATE
- DENSIFY
- ELU
- EMBEDDING_LOOKUP_SPARSE
- FAKE_QUANT
- FLOOR_DIV
- FLOOR_MOD
- GATHER_ND
- HARD_SWISH
- LEAKY_RELU
- LOG_SOFTMAX
- LSH_PROJECTION
- MATRIX_DIAG
- MATRIX_SET_DIAG
- MIRROR_PAD
- NON_MAX_SUPPRESSION_V4
- NON_MAX_SUPPRESSION_V5
- PADV2
- QUANTIZE
- RANK
- RELU_N1_TO_1
- RESIZE_NEAREST_NEIGHBOR
- REVERSE_SEQUENCE
- SCATTER_ND
- SEGMENT_SUM
- SELECT_V2
- SKIP_GRAM
- SPARSE_TO_DENSE
- SPLIT_V
- SQUARE
- SVDF
- UNIDIRECTIONAL_SEQUENCE_LSTM
- UNIDIRECTIONAL_SEQUENCE_RNN
- UNIQUE
- WHERE
- BCQ_FULLY_CONNECTED
- BCQ_GATHER

**Comment by seanshpark** [comment]
Closing as #2715 posted

**Pull request closed without merging**